### ISSUE TITLE
FEATURE: Join as Observer

### ISSUE THREAD

**Issue by justinleebehnke** [justinleebehnke/sheephead2#33]
The host should be able to allow observers to join a game as an observer.

The only thing that an observer can see are the cards in the current trick, and any other information that is already available to all of the players. Current Turn, Dealer, Picker, etc...

They should view the table as though they were sitting in the hosts chair (this doesn't really matter but they need to be able to view to order as someone and so it might as well be the host)

(There should be no reason for a player in the game to want to observe their own game in order to gather more information)

**Comment by Econgineer** [comment]
Maybe joining as observer allows seeing everyone's hand? Watching world series of poker without seeing the cards would totally stink.

**Comment by justinleebehnke** [comment]
Yes it would be better as long as it announced that someone is observing like you suggested.
